docs/main/curves.md

```diff
@@ -1,85 +1,90 @@
 
-### Supply
+2. **Capacity profiles**. These profiles are typically used for supply profiles (such as solar PV and wind) and represent the fraction of installed capacity used for each hour of the year for a specific production technology. For example, a value of 0.5 means that 50% of peak capacity is utilised in that hour. The sum of all hours should equal the total annual full load hours of that technology. Note that a capacity profile is also used for the must-run interconnectors import and export profiles.
 
-|Sector|Source|Method|
-|---|---|---|
-|Solar PV|["Open Power System Data platform"](https://data.open-power-system-data.org) |Profile from measured data, adjusted to match country specific full load hours|
-|Solar Thermal|["KNMI"](https://knmy.readthedocs.io/en/latest/) |Profile from measured data, adjusted to solar-thermal behaviour|
-|Wind|["Open Power System Data platform"](https://data.open-power-system-data.org) and [Renewables.ninja](https://www.renewables.ninja/) |OPSD: Profile from measured data, adjusted to match country specific full load hours. Renewables.ninja: Modelled profiles available from 1980-present using satellite weather data. |
-|Other|Geothermal heat, geothermal power, hydro, biogas CHP, waste incinerator|Flat curve||
-|Dispatchable technologies||Production determined by [merit order](merit-order.md)|
+3. **Price profiles**. These profiles are used for the interconnector prices and represent the price for each hour of the year. The unit is €/MWh and the value for each hour will be rounded to the nearest whole cent. 
 
-### Time curves
-Time curves define how the national production of energy carriers changes over the years (up to 2040)
+4. **Availability profiles**. These profiles are used for interconnector import and export availability and represent the availability of the interconnector for each hour of the year. Values should be specified between 0 and 1 for each hour, where for example 0.5 means that 50% of the capacity is available, 0 that no capacity is available or 1 that all capacity is available.
 
-They are documented in on ETDataset in the source analyses of the specific datasets. [Example for the Netherlands - 2015](https://github.com/quintel/etdataset/blob/master/source_analyses/nl/2015/3_primary_production/3_primary_produciton_source_analysis.md)
+5. **Temperature curve**. Used to set the air temperature profile and represents the outdoor air temperature for each hour of the year. The unit is degrees Celsius.
 
-For the Netherlands the time curves are based on:
+## Overview of profiles
```

Comment:
I think it would be good to add a reference to ETDataset here. Generic information about profiles can be found on the documentation, specific sources per country on ETDataset.